crates/egui/src/containers/resize.rs

```diff
@@ -296,7 +291,7 @@ impl Resize {
             content_ui,
         } = prepared;
 
-        state.last_content_size = content_ui.min_size();
+        state.last_content_size = content_ui.min_size().at_least(state.last_content_size);
```

Comment:
To prevent `egui::Window` from growing or shrinking, it maintains the size it had when it was at its largest size.